src/common/exceptions/http/conflict.http.exception.ts

```diff
@@ -0,0 +1,16 @@
+import { LogContext, AlkemioErrorStatus } from '@common/enums';
+import { BaseHttpException } from './base.http.exception';
+import { HttpStatus } from '@nestjs/common';
+import { ExceptionDetails } from '@common/exceptions/exception.details';
+
+export class ConflictHttpException extends BaseHttpException {
+  constructor(
+    message: string,
+    context: LogContext,
+    code: AlkemioErrorStatus | string = AlkemioErrorStatus.UNSPECIFIED,
```

Comment:
If it is a conflict, the error status should be that it is a conflict, I don't get the logic here.